CDP4Composition/Exceptions/SettingsException.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// -------------------------------------------------------------------------------------------------
-// <copyright file="PluginSettingsException.cs" company="RHEA System S.A.">
-//   Copyright (c) 2019 RHEA System S.A.
+// <copyright file="SettingsException.cs" company="RHEA System S.A.">
```

Comment:
Please realize that by removing the `PluginSettingsExcption` the composition library is no longer forwards compatible with the current version, this is an API change which requires a major version update. I am not stating this is wrong, but you have to understand the consequences here.

Reply:
I agree with @samatrhea here. You can leave `PluginSettingsException` and instead create an abstract parent `SettingsException` and another derived `AppSettingsExcetion`